src/main/resources/application.yml

```diff
@@ -46,3 +46,10 @@ management:
 
 springdoc:
   version: "@project.version@"
+
+site:
+  url:
+    base: ${siteUrlBase}
```

Comment:
siteUrlBase translates to nldiProtocol, nldiHost, and nldiPath in this app, so lines 51 & 52 are not needed.

deployName is not used/needed for this app, so line 55 is not needed.

apiDocsUrl has been hard coded in the ConfigurationService, so line 54 is not needed.

Since everything under it is gone, line 50 is not needed either.